### ISSUE TITLE
Look like Crestapp not support  Pgsql when generate from database.

### ISSUE THREAD

**Issue by ninjadev129** [CrestApps/laravel-code-generator#203]
**Is your feature request related to a problem? Please describe.**

**Describe the solution you'd like**

**Describe alternatives you've considered**

**Additional context**

**Comment by MikeAlhayek** [comment]
Yes. Creating resources from the database is only supported for MySql. If you like to contribute, feel free to submit a PR for other RDBMS.
